<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHiMDUploadDialog</class>
 <widget class="QDialog" name="QHiMDUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track upload - QHiMDTransfer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/upload_from_md.png</normalon>
   </iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QWidget" name="current" native="true">
     <layout class="QVBoxLayout" name="currentlayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="overall">
        <item>
         <widget class="QLabel" name="OverallProgress">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>overall progress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="AllPBar"/>
        </item>
        <item>
         <widget class="QLabel" name="ETA">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="TrkPBar"/>
        </item>
        <item>
         <widget class="QLabel" name="curtrack_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Summary">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonbox">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <widget class="QPushButton" name="details_button">
            <property name="text">
             <string>Show Details</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_button">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_button">
            <property name="text">
             <string>Close</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="details">
     <item>
      <widget class="QTreeWidget" name="ErrorList">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>80</number>
       </attribute>
       <column>
        <property name="text">
         <string>Tracknr.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Error message</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
